schedule/sle-perf/hanaperf_installation.yaml

```diff
@@ -0,0 +1,59 @@
+---
+name: hanaperf_installation
+description: >
+  OS installation for HANA performance
+vars:
+  MULTIPATH: '1'
+  MULTIPATH_CONFIRM: 'yes'
+  SEPARATE_HOME: '1'
+  NUMDISKS: '7'
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - '{{scc_registration}}'
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_smalldisk_storageng
+  - '{{separate_home}}'
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - boot/reconnect_mgmt_console
+  - support_server/login
+  - kernel_performance/install_qatestset
+  - boot/reconnect_mgmt_console
+  - support_server/login
+  - kernel_performance/full_run
+conditional_schedule:
+  scc_registration:
+    SCC_REGISTER:
+      none:
+        - '{{multipath}}'
+        - installation/scc_registration
+      installation:
+        - installation/scc_registration
+        - '{{multipath}}'
+  multipath:
+    MULTIPATH:
+      1:
+        - installation/multipath
+  separate_home:
+    SEPARATE_HOME:
+      1:
+        - installation/partitioning/no_separate_home
+        - '{{enlarge_swap}}'
+  enlarge_swap:
+    VERSION:
+      15-SP2:
+        - installation/partitioning/no_enlarge_swap
+      0:
```

Comment:
Correct me if I'm wrong, but as far as I understand this condition means:

If `VERSION=15-SP2` then schedule `no_enlarge_swap` test module, but in case `VERSION=0` - schedule `enlarge_swap` module.

As far as I know, we don't have any test suite with `VERSION=0`. So, this condition will never be true. Maybe you wanted to do something different, or I did not get something?:)

Reply:
Removed VERSION=0 and enlarge_swap.pm